frontend/index.js

```diff
@@ -5,21 +5,28 @@ let go = document.getElementById("goGenerator")
 
 let editor = ""
 
-window.generatorCall=function (){
-  let yamlData  = document.getElementById("yamlGenerator").value
+window.generatorCall = function () {
+  let yamlData = document.getElementById("yamlGenerator").value
   document.getElementById('yamlGenerator').style.border = "1px solid #ced4da"
   yamlData = editor.getValue()
   $.ajax({
+<<<<<<< HEAD
+    'url': `${URL}`,
+    'type': 'POST',
+    'data': yamlData,
+    'success': function (data) {
+      go.setValue(data)
+=======
```

Comment:
@bajran Please resolve the conflicts
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github